### PR TITLE
Fix english typos and grammar in documentation

### PR DESCRIPTION
**Pull request by arpitjain099**
Documentation can use some typo fixes and grammar fixes in documentation. This PR fixes some of the issues. Please comment if you'd like me to look at more files

### CODE REVIEW

**Comment by Court72** [comment]
@arpitjain099 - It looks like a stray config file made its way into your working branch. Can you remove the **.vscode/settings.json** file from this PR?

**Comment by Court72** [comment]
@batamig, @dcurwin 

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: CloudAppSecurityDocs/access-policy-aad.md [hunk 1]
```diff
@@ -37,19 +37,19 @@ To create a new access policy, follow this procedure:
 
     - **Device tags**: Use this filter to identify unmanaged devices.
 
-    - **Location**: Use this filter to identify unknown (and therefore risky) locations.
+    - **Location**: This filter identifies unknown (and therefore risky) locations.
```
Comment (Court72) [inline CloudAppSecurityDocs/access-policy-aad.md:40]: This change makes the language in the list inconsistent. Consider leaving as-is or changing lines 38 and 42 to follow the "This filter..." construction as well.

File: CloudAppSecurityDocs/access-policy-aad.md [hunk 2]
```diff
@@ -37,19 +37,19 @@ To create a new access policy, follow this procedure:
...
 1. Under **Actions**, select one of the following options:
 
-    - **Test**: Set this action to explicitly allow access according to the policy filters you set.
+    - **Test**: Set this action to allow access according to the policy filters you set explicitly.
 
-    - **Block**: Set this action to explicitly block access according to the policy filters you set.
+    - **Block**: Set this action to block access according to the policy filters you set explicitly.
 
-1. You can **Create an alert for each matching event with the policy's severity** and set an alert limit and select whether you want the alert as an email, a text message or both.
+1. You can **Create an alert for each matching event with the policy's severity** and set an alert limit, and select whether you want the alert as an email, a text message, or both.
```
Comment (Court72) [inline CloudAppSecurityDocs/access-policy-aad.md:52]:
```suggestion
1. You can **Create an alert for each matching event with the policy's severity**, set an alert limit, and select whether you want the alert as an email, a text message, or both.
```

File: CloudAppSecurityDocs/accounts.md [hunk 3]
```diff
@@ -8,60 +8,60 @@ ms.topic: how-to
...
 
-Admins can search for a specific user's metadata or user's activity. The **Users and accounts** page provides you with comprehensive details about the entities that are pulled from connected cloud applications. It also provides the user's activity history and security alerts related to the user.
+Admins can search for a specific user's metadata or user's activity. The **Users and accounts** page provide you with comprehensive details about the entities that are pulled from connected cloud applications. It also provides the user's activity history and security alerts related to the user.
```
Comment (Court72) [inline CloudAppSecurityDocs/accounts.md:13]:
```suggestion
Admins can search for a specific user's metadata or user's activity. The **Users and accounts** page provides you with comprehensive details about the entities that are pulled from connected cloud applications. It also provides the user's activity history and security alerts related to the user.
```

File: CloudAppSecurityDocs/accounts.md [hunk 4]
```diff
@@ -8,60 +8,60 @@ ms.topic: how-to
...
 
-The **Users and accounts** page can be [filtered](#users-and-accounts-filters) to enable you to find specific accounts and to deep dive into different types of accounts, for example, you can filter for all External accounts that haven't been accessed since last year.
+The **Users and accounts** page can be [filtered](#users-and-accounts-filters) to enable you to find specific accounts and to deep dive into different types of accounts; for example, you can filter for all External accounts that haven't been accessed since last year.
 
-The **Users and accounts** page enables you to easily investigate your accounts, including the following issues:
+The **Users and accounts** page enables you to investigate your accounts, including the following issues easily:
```
Comment (Court72) [inline CloudAppSecurityDocs/accounts.md:17]:
```suggestion
The **Users and accounts** page enables you to easily investigate your accounts, including the following issues:
```

File: CloudAppSecurityDocs/accounts.md [hunk 5]
```diff
@@ -8,60 +8,60 @@ ms.topic: how-to
...
 
 * Check if any accounts have been inactive in a particular service for a long time (Maybe you should revoke the license for that user to that service)
 
 * You can filter for the list of users with admin permissions
 * You can search for users who are no longer part of your organization but may still have active accounts
-* You can take [governance actions](#governance-actions) on the accounts such as suspend an app or go to the account settings page.
+* You can take [governance actions](#governance-actions) on the accounts, such as suspending an app or going to the account settings page.
```
Comment (Court72) [inline CloudAppSecurityDocs/accounts.md:23]:
```suggestion
* You can take [governance actions](#governance-actions), such as suspending an app or going to the account settings page, on the accounts.
```

File: CloudAppSecurityDocs/activity-filters-queries.md [hunk 6]
```diff
@@ -72,30 +72,30 @@ Below is a list of the activity filters that can be applied. Most filters suppor
...
   - User group – Specific user groups that you can import from connected apps, for example, Office 365 administrators.
-  - User name - Search for a specific username. To see a list of users in a specific user group, in the **Activity drawer**, select the name of the user group. Clicking will take you to the Accounts page that lists all the users in the group. From there, you can drill down into the details of the accounts of specific users in the group.
-  - The **User group** and **User name** filters can be further filtered by using the **As** filter, and selecting the role of the user, which can be any of the following:
-    - Activity object only - meaning that the user or user group selected didn't perform the activity in question, they were the object of the activity.
+  - User name - Search for a specific username. To see a list of users in a specific user group, in the **Activity drawer**, select the name of the user group. Clicking will take you to the Accounts page, that lists all the users in the group. From there, you can drill down into the details of the accounts of specific users in the group.
```
Comment (Court72) [inline CloudAppSecurityDocs/activity-filters-queries.md:87]:
```suggestion
  - User name - Search for a specific username. To see a list of users in a specific user group, in the **Activity drawer**, select the name of the user group. Clicking will take you to the Accounts page, which lists all the users in the group. From there, you can drill down into the details of the accounts of specific users in the group.
```

File: CloudAppSecurityDocs/activity-filters-queries.md [hunk 7]
```diff
@@ -117,27 +117,27 @@ To make investigation even simpler, you can now create custom queries and save t
...
+Defender for Cloud Apps also provides you with **Suggested queries**. Suggested queries provide you with recommended avenues of investigation that filter your activities. You can edit these queries and save them as custom queries. The following are optional suggested queries:
 
-- Admin activities - filters all your activities to display only those activities that involved admins.
+- Admin activities - filters all your activities to display only those activities that involve admins.
 
-- Download activities - filters all your activities to display only those activities that were download activities, including download user list as a .csv file, downloading of shared content, and downloading a folder.
+- Download activities - filters all your activities to display only those activities that were download activities, including downloading user list as a .csv file, downloading shared content, and downloading a folder.
 
-- Failed log in - filters all your activities to display only failed log-ons and failed sign in via SSO
+- Failed log-in - filters all your activities to display only failed logons and failed signs in via SSO
```
Comment (Court72) [inline CloudAppSecurityDocs/activity-filters-queries.md:126]:
```suggestion
- Failed log-in - filters all your activities to display only failed log-ins and failed sign-ins via SSO
```

File: CloudAppSecurityDocs/access-policy-aad.md (hunk 1 above)
Comment (batamig) [inline CloudAppSecurityDocs/access-policy-aad.md:40]: agree with @Court72 - reverting changes to these lines.
```suggestion
    - **Location**: Use this filter to identify unknown (and therefore risky) locations.
```

File: CloudAppSecurityDocs/access-policy-aad.md [hunk 8]
```diff
@@ -37,19 +37,19 @@ To create a new access policy, follow this procedure:
...
     - **Device tags**: Use this filter to identify unmanaged devices.
 
-    - **Location**: Use this filter to identify unknown (and therefore risky) locations.
+    - **Location**: This filter identifies unknown (and therefore risky) locations.
 
-    - **IP address**: Use this filter to filter per IP addresses or use previously assigned IP address tags.
+    - **IP address**: Use this filter to filter per IP address or use previously assigned IP address tags.
 
-    - **User agent tag**: Use this filter to enable the heuristic to identify mobile and desktop apps. This filter can be set to equals or does not equal. The values should be tested against your mobile and desktop apps for each cloud app.
+    - **User agent tag**: This filter enables the heuristic to identify mobile and desktop apps. This filter can be set to equal or not equal. The values should be tested against your mobile and desktop apps for each cloud app.
```
Comment (batamig) [inline CloudAppSecurityDocs/access-policy-aad.md:44]:
```suggestion
    - **User agent tag**: Use this filter to enable the heuristic to identify mobile and desktop apps. This filter can be set to equal or not equal. The values should be tested against your mobile and desktop apps for each cloud app.
```

File: CloudAppSecurityDocs/accounts.md (hunk 5 above)
Comment (batamig) [reply CloudAppSecurityDocs/accounts.md:23]: the original PR change seems better to me, I'm choosing to keep the original PR change.

File: CloudAppSecurityDocs/activity-privacy.md [hunk 9]
```diff
@@ -52,14 +52,14 @@ To configure your user privacy groups, you must first [import user groups](user-
...
 
 ### To view private activities
 
-1. In the **Activity log** page, to the right of the activity table, click the settings icon, and then select **Show private activities**.
+1. In the **Activity log** page, click the settings icon to the right of the activity table, and then select **Show private activities**.
 
     ![Screenshot showing the activity log settings icon.](media/activity-privacy-view-settings-icon.png)
 
-1. In the **Show private activities** dialog, click **OK** to confirm that you understand that the action is being audited. Once confirmed, the private activities are shown in the activity log and the action is recorded in the governance log.
+1. In the **Show private activities** dialog, click **OK** to confirm that you understand the action is being audited. Once confirmed, the private activities are shown in the activity log, and the action is recorded in the governance log.
```
Comment (batamig) [inline CloudAppSecurityDocs/activity-privacy.md:63]: this seems to change the meaning of the phrase; reverting.
```suggestion
1. In the **Show private activities** dialog, click **OK** to confirm that you understand that the action is being audited. Once confirmed, the private activities are shown in the activity log, and the action is recorded in the governance log.
```

**Comment by batamig** [comment]
Hi @arpitjain099 - thanks for your contributions to docs! We've reviewed the PR and made some slight changes and will be merging shortly.

**Comment by batamig** [comment]
#sign-off

**Pull request merged**